local_gemma/modeling_local_gemma_2.py

```diff
@@ -134,10 +135,14 @@ def from_pretrained(
             preset_kwargs["quantization_config"] = quantization_config
         elif preset_kwargs.get("quantization_config"):
             if device == "cuda":
-                preset_kwargs["quantization_config"] = BitsAndBytesConfig(
-                    load_in_4bit=True,
-                    bnb_4bit_compute_dtype=preset_kwargs["torch_dtype"],
-                )
+                if quantization == "hqq":
+                    preset_kwargs["quantization_config"] = HqqConfig(nbits=4, group_size=64, quant_zero=False, quant_scale=False, axis=1) 
+                    preset_kwargs["device_map"] = "cuda"
```

Comment:
hqq not compatible with cpu offload? the only reason we set `device_map="cuda"` here?